test/integration/forms/form.js

```diff
@@ -1234,35 +1234,19 @@ context('Patient Action Form', function() {
         fx.data.id = '1';
         fx.data.relationships.form.data = { id: '11111' };
         fx.data.relationships['program-action'] = { data: { id: '11111' } };
-        fx.data.relationships['form-responses'].data = [
-          { id: '1', meta: { created_at: testTs() } },
-        ];
```

Comment:
Removed this and `routeFormResponse()` as they weren't necessary for this test. And allows us to not show the `Update` button initially.